php-frontend/src/main/java/org/sonar/php/ini/PhpIniParser.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.php.ini;
+
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.List;
+import org.sonar.php.ini.tree.Directive;
+import org.sonar.php.ini.tree.PhpIniFile;
+import org.sonar.php.tree.impl.lexical.InternalSyntaxToken;
+import org.sonar.plugins.php.api.tree.lexical.SyntaxToken;
+
+public class PhpIniParser {
+
+  public PhpIniFile parse(String content) {
+    List<Directive> directives = new ArrayList<>();
+    String lines[] = content.split("\\r?\\n");
+    int lineNumber = 1;
+    for (String line : lines) {
+      Directive directive = parseLine(line, lineNumber);
+      if(directive != null) {
+        directives.add(directive);
+      }
+      lineNumber++;
+    }
+    return new PhpIniFileImpl(directives);
+  }
+
+  private static Directive parseLine(String line, int lineNumber) {
+    int equalSignIndex = 0;
+    boolean insideString = false;
+    boolean afterBackSlash = false;
+
+    int i = 0;
+    for (; i < line.length(); i++) {
+      char c = line.charAt(i);
+      if (c == '=') {
+        equalSignIndex = i;
+      }
+      if (c == ';' && !insideString) {
+        break;
+      }
+      if (c == '"' && !afterBackSlash) {
+        insideString = !insideString;
+      }
+      afterBackSlash = c == '\\';
+    }
+
+    String name = line.substring(0, equalSignIndex);
+    if (name.isEmpty()) {
+      return null;
```

Comment:
@pynicolas In fact we don't create directive when `=` starts the line, but we do when name contains only whitespaces. It doesn't look consistent